src/PKSim.Core/Commands/SetExpressionProfileValueCommand.cs

```diff
@@ -38,7 +38,7 @@ protected override void UpdateDependenciesOnParameter(IParameter parameter, IExe
 
       protected override ICommand<IExecutionContext> GetInverseCommand(IExecutionContext context)
       {
-         return new SetExpressionProfileValueCommand(_parameter, _oldValue).AsInverseFor(this);
+         return new SetExpressionProfileValueCommand(_parameter, _oldValue, _updateSimulationSubjects).AsInverseFor(this);
```

Comment:
this was probalby wrong here since we were not passing the parameter when doing a rollback (only a problem when false as default is true=> just takes longer)